src/embind/embind.js

```diff
@@ -76,7 +76,7 @@ var LibraryEmbind = {
     }
     types.forEach(visit);
 
-    throw new UnboundTypeError(message + ': ' + unboundTypes.map(getTypeName).join([', ']));
+    throw new UnboundTypeError(message + ': ' + unboundTypes.map(___getTypeName).join([', ']));
```

Comment:
I don't think this is the right fix. I think the right fix is to add ``$getTypeName`` to the deps of ``$throwUnboundTypeError``